// Fails due to thrown error.

throw Error('Oops')
